code/index.html

```diff
@@ -1,64 +1,63 @@
-  <body>
-    <aside class="question-section">
-      <button id="restart" class="outlined-button">RESTART</button>
 
-      <img
-        class="guess-who-icon"
-        src="images/guess-who-bubble.png"
-        alt="Guess Who"
-      />
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Guess Who</title>
+  <link rel="stylesheet" href="style.css" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet" />
+</head>
 
-      <h1>Does the person have</h1>
+<body>
+  <aside class="question-section">
+    <button id="restart" class="outlined-button">RESTART</button>
 
-      <select id="questions">
-        <optgroup label="hair color">
-          <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
-        </optgroup>
-        <optgroup label="eye color">
-          <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
-        </optgroup>
-        <optgroup label="accessories">
-          <option value="glasses">glasses</option>
-          <!-- Add the other accessory option here. (hat)  -->
-        </optgroup>
-        <optgroup label="other">
-          <option value="smoker">a smoking habit</option>
-        </optgroup>
-      </select>
+    <img class="guess-who-icon" src="images/guess-who-bubble.png" alt="Guess Who" />
 
-      <button id="filter" class="filled-button">FIND OUT</button>
-    </aside>
+    <h1>Does the person have</h1>
 
-    <section class="board-wrapper">
+    <select id="questions">
+      <optgroup label="hair color">
+        <option value="hidden">hidden</option>
+        <option value="black">black hair</option>
+        <option value="brown">brown hair</option>
+        <option value="grey">grey hair</option>
+        <option value="orange">orange hair</option>
+        <option value="purple">purple hair</option>
+        <option value="yellow">yellow hair</option>
+        <option value="white">white hair</option>
+
+      </optgroup>
+      <optgroup label="eye color">
+        <option value="hidden">hidden eyes</option>
+        <option value="blue">blue eyes</option>
+        <option value="brown">brown eyes</option>
+        <option value="green">green eyes</option>
+
+      </optgroup>
+      <optgroup label="accessories">
+        <option value="glasses">glasses</option>
+        <option value="hat">hat</option>
+
+      </optgroup>
+      <optgroup label="other">
+        <option value="smoker">a smoking habit</option>
+      </optgroup>
+    </select>
+
+    <button id="filter" class="filled-button">FIND OUT</button>
+    <div class="guess-counter" id="guess-counter" </div> </aside> <section class="board-wrapper">
```

Comment:
I didn't know about the </aside> tag and googled it to find out what it does haha. 
Have been trying to figure out what it does specifically in your code, but it could just be me not seeing it!